internal/broker/bind_create_test.go

```diff
@@ -171,21 +173,21 @@ func TestCreateBindingEndpoint(t *testing.T) {
 		nil,
 		bindEndpoint,
 		nil,
-		nil,
+		getBindingEndpoint,
 		nil,
 	}, brokerLogger)
 
 	//// attach bindings api
-	method := "PUT"
 	router := mux.NewRouter()
-	router.HandleFunc("/v2/service_instances/{instance_id}/service_bindings/{binding_id}", apiHandler.Bind).Methods(method)
+	router.HandleFunc("/v2/service_instances/{instance_id}/service_bindings/{binding_id}", apiHandler.Bind).Methods("PUT")
```

Comment:
Does it mean that by `bind` you mean `create binding` (using `PUT`), by `getBinding` you mean `getBinding` (using `GET`)?

Reply:
I have not come up with those names. Let's discuss refactoring in following PRs.